colcon_cmake/task/cmake/__init__.py

```diff
@@ -215,24 +215,22 @@ def get_visual_studio_version():
 """
 Global variable for the cached CMake version number.
 
-When valid, this will be of pkg_resources.extern.packaging.version.Version
-It may also have a boolean value False when the CMake version has failed
-to parse.
+When valid, this will be a pkg_resources.extern.packaging.version.Version.
+It may also have a boolean value False when the CMake version could not be
+determined to avoid trying to determine it again.
 """
 _cached_cmake_version = None
 
 
 def get_cmake_version():
     """
-    Get the cached CMake version number if available and successfully parsed.
+    Get the CMake version.
```

Comment:
I agree. Currently the additional information if in the following paragraph. Please feel free to suggest a better first sentence (which passes the linter in terms of line length).

Reply:
If I'm right with the line limit being 80 characters, then the orignal comment fit. Otherwise, how about:

> Query the CMake --version results if available.

Otherwise, I'm not going to deliberate over this one. I'd like to complete the PR.

Reply:
I wouldn't use "query". Also "results" doesn't sound right to me. And "if available" is fuzzy too.
How about:

> Get the version as reported by `CMAKE_EXECUTABLE --version` if parsable.

The redundant detail can then be removed from `:returns:`:

> The CMake version, or None when the version number could not be determined